plugins/packages/zendesk/lib/manifest.json

```diff
@@ -0,0 +1,42 @@
+
+{
+  "$schema": "https://raw.githubusercontent.com/ToolJet/ToolJet/develop/plugins/schemas/manifest.schema.json",
+  "title": "Zendesk datasource",
+  "description": "A schema defining Zendesk datasource",
+  "type": "api",
+  "source": {
+    "name": "Zendesk",
+    "kind": "zendesk",
+    "exposedVariables": {
+      "isLoading": false,
+      "data": {},
+      "rawData": {}
+    },
+    "options": {},
+    "customTesting": true,
+    "hideSave": true
+  },
+  "defaults": {
+    "access_type": {
+      "value": "read"
+    }, 
+    "subdomain": {
+      "value": ""
+    },
+    "client_id": {
+      "value": ""
+    },
+    "client_secret": {
+      "value": ""
+    }
+  },
+  "properties": {
+    "zendesk": {
+      "label": "",
+      "key": "zendesk",
+      "type": "react-component-zendesk",
```

Comment:
Q: would it not be possible to render this using dynamic form or alter dynamic form to in a certain way to handle zendesk's usecase?

Reply:
Yes. Let's do it in a separate PR and include gsheet and slack form.